ckanext/stadtzhtheme/commands.py

```diff
@@ -125,3 +155,70 @@ def _get_datastore_table_page(self, context, offset=0):
         has_next_page = (len(result['records']) > 0)
 
         return (resource_id_list, has_next_page)
+
+    def _cleanup_storage_files(self, context, resource_path):
+        files_to_delete = []
+        files_to_keep = []
+        for root, dirs, files in os.walk(resource_path, topdown=True):
+            if files:
+                for file in files:
+                    resource_id = ''.join(root.split('/')[-2:]) + file
+                    file_path = os.path.join(root, file)
+                    try:
+                        logic.check_access('resource_show', context)
+                        logic.get_action('resource_show')(
+                            context,
+                            {'id': resource_id}
+                        )
+                        files_to_keep.append(file_path)
+                    except logic.NotFound:
+                        files_to_delete.append(file_path)
+                    except logic.NotAuthorized:
+                        print("User is not authorized to perform this action.")
+                        sys.exit(1)
+                    except (KeyError, AttributeError), e:
+                        raise ("Error while handling record {}: {}"
+                               .format(resource_id, str(e)))
+        print("Remove files from storage where resource has been deleted:\n"
```

Comment:
Can we change this to `{} files should be deleted` or similar? Otherwise, I find it's misleading.